src/wallet/rpcwallet.cpp

```diff
@@ -3358,62 +3358,6 @@ static UniValue bumpfee(const JSONRPCRequest& request)
     return result;
 }
 
-UniValue generate(const JSONRPCRequest& request)
```

Comment:
I expect you can also remove the `#include <rpc/mining.h>` include from this file.

Reply:
Done